### ISSUE TITLE
<Trans id={'msg.id'} /> does not work

### ISSUE THREAD

**Issue by HriBB** [lingui/js-lingui#188]
CLI extract parser does not handle attributes wrapped in curly brackets.

This works
```
<Trans id="msg.id" />
```
This works
```
<Trans id='msg.id' />
```
This does not work
```
<Trans id={'msg.id'} />
```

I can make a PR if you'd like, just point me in the right direction.

I assume we need to add support in `babel-plugin-extract-messages`, right?

**Comment by HriBB** [comment]
I created a #189 for this issue.

**Comment by HriBB** [comment]
I'm trying to test this locally, but I cannot get the cli to work. Is there some documentation on contributing or how to setup the dev env?

**Comment by machour** [comment]
@HriBB there's a contributing guide at https://github.com/lingui/js-lingui/blob/master/CONTRIBUTING.md

**Comment by tricoder42** [comment]
Hey @HriBB, thank you for the PR.

I'm wondering, what's the usecase? ID should be always either a string or a variable (inside JSX container). String is generated automatically in transform plugins. What about raising a warning about unnecessary JSX container when user accidentally types it?

**Comment by HriBB** [comment]
I find it faster to refactor and change props, if I use this format all the time `<Component prop={'string'} />`.

So when integrating `js-lingui` in one of my apps, I used `<Trans id={'msg.id'} />` in some places, and those messages were not extracted. After changing it to `<Trans id="msg.id'"/>`, all messages are extracted correctly.

It's not a problem, but might be worth mentioning in the docs. I made the PR just to learn a little bit about babel. If you want, you can close it ;)

**Comment by tricoder42** [comment]
That's fine, sounds good. Thanks for clarifying! 👍

**Comment by tricoder42** [comment]
Fixed in *2.1.2*, thanks!
